pages/admin/interviewer/invite.tsx

```diff
@@ -0,0 +1,256 @@
+      })
+    }
+
+    reader.readAsBinaryString(file)
+  }
+
+  const sendReminders = () => {
+    if (loading) return
+
+    setLoading(true)
+    unresposiveInterviewers.forEach((Interviewer, index) => {
+      setTimeout(() => {
+        sendIntReminder(Interviewer.email, Interviewer.reminder + 1 || 1)
+
+        if (index === unresposiveInterviewers.length - 1) {
+          setTimeout(() => {
+            setLoading(false)
+            alert('Sent reminder to unresponsive Interviewers!')
+          }, 1000)
+        }
+      }, 1000 * index)
+    })
+  }
+
+  return (
+    <AdminLayout>
+      <div className="py-10">
+        <h1 className="text-sm sm:text-xl md:text-2xl bg-blue-850 my-10 text-white text-center font-extrabold py-2 rounded-tl-3xl rounded-br-3xl">
+          Invite Interviewers
+        </h1>
+
+        <div className="flex justify-around mt-20 mb-10">
+          <input
+            type="file"
+            accept=".xlsx"
+            onChange={(e) => setFile(e.target.files[0])}
+            className="text-white text-base font-bold md:text-lg bg-gray-500 rounded-lg cursor-pointer w-max"
+          />
+          <button
+            onClick={proceed}
+            className="text-white text-base md:text-lg bg-blue-850 ml-2 px-5 rounded-lg flex flex-row items-center cursor-pointer"
+          >
+            {!loading ? (
+              'Submit'
+            ) : (
+              <FontAwesomeIcon
+                icon={faSpinner}
+                width={30}
+                spin={true}
+                className="mx-5"
+              />
+            )}
+          </button>
+        </div>
+        <p className="text-red-850 text-center mb-10">
+          Note: Provide the excel file containing the data in the specified
+          format.
+        </p>
+
+        <div>
+          <h2 className="text-sm sm:text-lg md:text-xl bg-blue-850 my-10 text-white text-center font-extrabold py-2 rounded-tl-3xl rounded-br-3xl">
+            Accepted Interviewers
+          </h2>
+
+          <div className="flex justify-center">
+            <table className="border-separate p-2">
+              <thead>
+                <tr>
+                  <th className="border border-blue-850 p-2">S.No.</th>
+                  <th className="border border-blue-850 p-2">Name</th>
+                  <th className="border border-blue-850 p-2">Email</th>
+                </tr>
+              </thead>
+              <tbody>
+                {acceptedInterviewers.map((Interviewer, index) => (
+                  <tr key={index}>
+                    <td className="border border-blue-850 p-2">{index + 1}</td>
+                    <td className="border border-blue-850 p-2">
+                      {Interviewer.name}
+                    </td>
+                    <td className="border border-blue-850 p-2">
+                      {Interviewer.email}
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
```

Comment:
This table code is similar for all accepted, rejected and unresponsive interviewers.
So create a dynamic component for this table by using users list as props. Use this component for each interviewers and reviewers list in their invite pages.